DangProject/Sources/Presentation Layer/Setting/Alarm/View/AlarmDaySelectionView.swift

```diff
@@ -33,58 +32,79 @@ class AlarmDaySelectionView: UIView {
     var thursdayButton = UIButton()
     var fridayButton = UIButton()
     var saturdayButton = UIButton()
-    var tagNumber = 0
-    var delegate: AlarmDaySelectionDelegate?
-    
-    
+
     override init(frame: CGRect) {
         super.init(frame: frame)
-        if #available(iOS 15.0, *) {
             configureUI()
-        } else {
-            // Fallback on earlier versions
-        }
     }
     
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    func configureButtonColor(_ tags: [Int]) {
```

Comment:
```suggestion
    func configureButtonColor(_ tags: [DayType]) {
```

이런 식으로 되면 당연히
잘 읽히고 마지막 default같은 것도 필요없게 될 것 같아요

Reply:
요일을 열거형으로 지정하고 rawValue 지정해줬습니다
좋은의견 감사합니다 ^^
e1016fcd4437c2b6aff96de5569834b16d18d99e